packages/text-area/src/vaadin-text-area-mixin.js

```diff
@@ -209,6 +219,36 @@ export const TextAreaMixin = (superClass) =>
       }
     }
 
+    /** @private */
+    __updateMaxHeight(maxRows) {
```

Comment:
Should we prepare for the app dynamically changing root em = calling `this.__updateMaxHeight(this.maxRows);` in  `_updateHeight`?

Reply:
It seems to work well with the change:
- Both text areas have max-rows of 4
- The first text area exceeds min-rows, the second one doesn't

Before:

https://github.com/user-attachments/assets/2a32035a-07aa-47d8-8d03-28f0920ea2b9

After:

https://github.com/user-attachments/assets/9e2f4080-3c54-4bd0-b8f2-0f4099d19927

Reply:
It probably works "by accident" in this case because root em also affects the component's label and other sizing values so ResizeObserver is triggered. Even if it didn't, it's probably better if it adjusts after the user starts typing than having a max-height calculated with an old value.

Reply:
Right, I was testing by just changing the font size of the textarea. But I agree, better to have it update in some cases than not at all.

Reply:
Done